src/_guides/language/language-tour.md

```diff
@@ -2473,18 +2475,17 @@ name) constructor.
 Use a named constructor to implement multiple constructors for a class
 or to provide extra clarity:
 
-<?code-excerpt "language-tour/reference/named_constructor.dart"?>
+<?code-excerpt "lib/language_tour/classes/point.dart (named-constructor)" replace="/Point\.\S*/[!$&!]/g"?>
 {% prettify dart %}
 class Point {
-  num x;
-  num y;
+  num x, y;
 
   Point(this.x, this.y);
 
   // Named constructor
-  Point.fromJson(Map json) {
-    x = json['x'];
-    y = json['y'];
+  [!Point.origin()!] {
+    x = 0;
+    y = 0;
```

Comment:
A slightly simpler named constructor example (we use the `Point.fromJson()` constructor elsewhere).